randovania/games/samus_returns/json_data/header.txt

```diff
@@ -11,7 +11,7 @@ Templates
       Bomb and Morph Ball
 
 * Lay Power Bomb:
-      Morph Ball and Power Bomb
+      Main Power Bomb and Morph Ball and Power Bomb
```

Comment:
This should not have the main power bomb listed

Reply:
For mains to be logical in the db it has to be. See dread for reference

Reply:
Dread does this "wrong". 
The way that mains work, is that until you get the main, you get a different item.
AM2R and echoes do this properly lemme see if i can find the conversation i had with dark about this.

Reply:
Found it:
- https://discord.com/channels/914291389293027329/1117068391858065509/1117846242710532198
- https://discord.com/channels/914291389293027329/1117068391858065509/1118222060670439578